cpp/basix/finite-element.cpp

```diff
@@ -952,7 +948,8 @@ FiniteElement<F>::FiniteElement(
     for (std::size_t col = 0; col < matM.extent(1); ++col)
     {
       F v = col == row ? 1.0 : 0.0;
-      if (std::abs(matM(row, col) - v) > 1.0e-12)
+      constexpr F eps = 100 * std::numeric_limits<F>::epsilon();
```

Comment:
Should this be `10.0 * ` and `<float>` to match the other line? Or are these intentionally different?

Reply:
Deliberate, but with without good justification ;).

I made the previous 'looser' because it follows a LU factorisation. I thought this check could be stricter. Proper 'fix' would be to support `float32` from Python and run the test suite, but we're not there yet.